_data/projects/.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: PyTorch
+desc: PyTorch is an open source machine learning library, used for applications such as computer vision and NLP. 
+site: https://github.com/pytorch/pytorch
+tags:
+- python
+- c++
+- machine learning
+- Cuda
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/up-for-grabs/up-for-grabs.net/labels/up-for-grabs
```

Comment:
This link is a URL for Up for Grabs itself - do you have a corresponding link for `pytorch/pytorch` that represents a list of issues that are ideal for new contributors?